src/orders/controller/order-detail.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller } from '@nestjs/common';
+import { ProductService } from 'src/products/service/product.service';
+import { OrderDetailMapper } from '../mapper/order-detail.mapper';
+import { OrderDetailService } from '../service/order-detail.service';
+
+@Controller('order-details')
+export class OrderDetailController {
+  private readonly orderDetailMapper: OrderDetailMapper;
+
+  constructor(
```

Comment:
I think this controller is not used right? You can remove it. We only want to keep used code.